pkg/sbom/edge.go

```diff
@@ -5,6 +5,12 @@ import (
 	"strings"
 )
 
+func NewEdge() *Edge {
+	return &Edge{
+		To: []string{},
+	}
+}
+
```

Comment:
I see "yay new function!" and then immediately think "ohh... need new test..." lol. I'll add edge_test.go after this is merged.